### PR TITLE
Strict filter for files deployed through styles

### PR DESCRIPTION
**Pull request by dtdesign**
n/t

### CODE REVIEW

File: wcfsetup/install/files/lib/data/style/StyleEditor.class.php [hunk 1]
```diff
@@ -42,6 +42,7 @@
 class StyleEditor extends DatabaseObjectEditor implements IEditableCachedObject {
 	const EXCLUDE_WCF_VERSION = '6.0.0 Alpha 1';
 	const INFO_FILE = 'style.xml';
+	const VALID_IMAGE_EXTENSIONS = ['.gif', '.jpg', '.jpeg', '.png'];
```
Comment (TimWolla) [inline wcfsetup/install/files/lib/data/style/StyleEditor.class.php]: svg and possibly webp?

Comment (dtdesign) [reply wcfsetup/install/files/lib/data/style/StyleEditor.class.php]: WebP isn't supported by us yet.

**Comment by jens1o** [comment]
Out of curiosity: Why don't you use `pathinfo`?

**Comment by dtdesign** [comment]
I was primarily reusing existing code, also using `pathinfo()` doesn't make a significant difference here.

**Pull request merged**